SConstruct

```diff
@@ -272,6 +275,63 @@ if useLz4:
 
 wt_objs = [env.Object(a) for a in wtsources]
 
+#
+# wiredtiger_java.dll
+## Usage example
+## c:\Python27\python.exe c:\Python27\Scripts\scons.py --enable-java=c:/Users/paco/Downloads/swigwin-3.0.12/swig.exe,"C:/Program Files/Java/jdk1.8.0_151/"
+useJava = GetOption("lang-java")
+if useJava:
+    useJavaSplit = useJava.split(',')
+    if len(useJavaSplit) == 2:
```

Comment:
It would be good to have an error reported if the option value doesn't have exactly one comma. For example, with `--enable-java=true` I think as written, it will silently continue and not build java.